mapall.py

```diff
@@ -93,9 +93,12 @@ def mn(self, s=None):
         """ Munge name to be dottable """
         if not s:
             s = self.name
-        s = s.replace('-', '_')
-        s = s.replace("'", '"')
-        return s
+        try:
+            s = s.replace('-', '_')
+            s = s.replace("'", '"')
+            return s
+        except Exception as e:
```

Comment:
We can fix this problem a few was, catch the exception or test for none.

If we opt for catching the exception, it would be better to have a tighter catch for just the AttributeError
```
>>> s = None
>>> s.replace('a', 'b')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'replace'
>>> 
```